Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -946,10 +947,12 @@ private void OnDestroy()
         [NonSerialized] internal InputActionRebindingExtensions.ParameterOverride[] m_ParameterOverrides;
 
         [NonSerialized] internal InputActionMap.DeviceArray m_Devices;
+        [SerializeField] internal int m_Version;
 
         [Serializable]
         internal struct WriteFileJson
         {
+            public int version;
```

Comment:
It's fine to have version here since related to the file. At the same time it introduces some technical weight to carry forward, but it makes sense to have a version defined unless it can be determined based on structure.